apps/expo/jest.config.ts

```diff
@@ -31,10 +31,13 @@ const config: Config = {
   //   },
   // },
   // preset: 'ts-jest',
+
+  /* DON'T UNCOMMENT THIs, IT GOES KABOOM */
```

Comment:
If we don't need all this code, can we just remove it instead of commenting it out? Or does it serve a purpose for the future?

Reply:
it might serve a purpose in the future, but do note that idk what it is for. The current config is good as it is.